src/overlays/actors/ovl_Demo_Kankyo/z_demo_kankyo.c

```diff
@@ -56,21 +56,21 @@ void DemoKakyo_LostWoodsSparkleActionFunc(DemoKankyo* this, PlayState* play) {
     Player* player = GET_PLAYER(play);
 
     if (play->roomCtx.unk7A[1] != 0) {
-        if (play->envCtx.unk_F2[3] != 0) {
-            play->envCtx.unk_F2[3]--;
+        if (play->envCtx.precipitation[PRECIP_SNOW_MAX] != 0) {
+            play->envCtx.precipitation[PRECIP_SNOW_MAX]--;
         } else {
             Actor_Kill(&this->actor);
         }
-    } else if (play->envCtx.unk_F2[3] < DEMOKANKYO_EFFECT_COUNT) {
-        play->envCtx.unk_F2[3] += 16;
+    } else if (play->envCtx.precipitation[PRECIP_SNOW_MAX] < DEMOKANKYO_EFFECT_COUNT) {
+        play->envCtx.precipitation[PRECIP_SNOW_MAX] += 16;
     }
 
     // note: DemoKankyo can crash if placed in an area that snows (ObjectKankyo)
     // because they both use unk_F2 as an effect counter,
     // causing DemoKankyo to write beyond its efffect array boundry
     // this crash can occur if the two actors are in different scenes connected by an exit
     // e.g. if you add DemoKankyo to GoronShrine, you will crash entering/leaving through door
-    for (i = 0; i < play->envCtx.unk_F2[3]; i++) {
+    for (i = 0; i < play->envCtx.precipitation[PRECIP_SNOW_MAX]; i++) {
```

Comment:
This comment could probably be updated with the change to `.precipitation[PRECIP_SNOW_MAX]`